apps/wpcom-block-editor/src/wpcom/features/tracking/wpcom-block-editor-global-styles-tab-selected.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * Internal dependencies
+ */
+import tracksRecordEvent from './track-record-event';
+
+const getTabNameFromId = ( id ) => ( id?.endsWith?.( '-block' ) ? 'block-type' : 'root' );
+
+export function trackGlobalStylesTabSelected( { tab, open } = {} ) {
+	const activeTabElement = document.querySelector(
+		'.edit-site-global-styles-sidebar .components-tab-panel__tabs button.is-active'
+	);
+	const activeTabName = tab || getTabNameFromId( activeTabElement?.id );
+	const isSidebarOpen =
+		typeof open !== 'undefined'
```

Comment:
just curious if the `typeof` approach buys us anything extra that `open !== undefined` wouldn't

Reply:
I never use `something !== undefined`. I find it a bad practice. It can cause errors in certain cases, like when a variable doesn't exist. `typeof` never fails though.